pkgs/top-level/all-packages.nix

```diff
@@ -10084,7 +10084,8 @@ in
 
   sourceFromHead = callPackage ../build-support/source-from-head-fun.nix {};
 
-  jruby = callPackage ../development/interpreters/jruby { };
+  jruby-group = callPackage ../development/interpreters/jruby {};
+  jruby = jruby-group.latest;
```

Comment:
this keeps all packages that rely on JRuby still working and backwards compatible